tests/src/testadaptivestepsizing.cpp

```diff
@@ -56,14 +57,17 @@ auto KLShellAndAdaptiveStepSizing(const PathFollowingType& pft, const std::vecto
   constexpr auto dimWorld        = 3;
   const std::array<int, 2> order = {2, 1};
 
-  const std::array<std::vector<double>, 2> knotSpans = {{{0, 0, 0, 1, 1, 1}, {0, 0, 1, 1}}};
+  const std::array<std::vector<double>, 2> knotSpans = {
+      {{0, 0, 0, 1, 1, 1}, {0, 0, 1, 1}}
+  };
 
   using ControlPoint = Dune::IGA::NURBSPatchData<2, dimWorld>::ControlPointType;
 
-  const std::vector<std::vector<ControlPoint>> controlPoints
-      = {{{.p = {-0.5, 0, 0.0}, .w = 10}, {.p = {-0.5, 1.0, 0.0}, .w = 1}},
-         {{.p = {0, 0, 0.2}, .w = 0.766044}, {.p = {0, 1.0, 0.2}, .w = 0.766044}},
-         {{.p = {0.5, 0, 0.0}, .w = 1}, {.p = {0.5, 1.0, 0.0}, .w = 1}}};
+  const std::vector<std::vector<ControlPoint>> controlPoints = {
+      {   {.p = {-0.5, 0, 0.0}, .w = 10},     {.p = {-0.5, 1.0, 0.0}, .w = 1}},
+      {{.p = {0, 0, 0.2}, .w = 0.766044}, {.p = {0, 1.0, 0.2}, .w = 0.766044}},
+      {     {.p = {0.5, 0, 0.0}, .w = 1},      {.p = {0.5, 1.0, 0.0}, .w = 1}}
+  };
```

Comment:
here the alignment of the control points looks nice but now the spacing of the `{` may be looks not that good, but I do not have any strong opinion if anything is to modified/discussed here. What do you think?

Reply:
If you are super unhappy with some code refactoring, you can also add 
```cpp
int formatted_code;
// clang-format off
    void    unformatted_code  ;
// clang-format on
void formatted_code_again;

```
in the future. For your given example i woulkd keep it for now.